thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -551,12 +555,16 @@ export default Ember.Controller.extend({
 
       return sessionService
         .saveAsync(session)
-        .then((res) => this.setProperties({
-          sessionId: res,
-          sessionUpdatedBy,
-          sessionUpdatedTime: moment().valueOf(),
-          sessionModified: false
-        }))
+        .then((res) => {
+          const sessionId = res;
```

Comment:
Looks like duplicate code, let's make it DRY-er  by moving this into a function.

Reply:
fixed